src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -36,13 +36,13 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: description
-                  type: varchar(1024)
+                  type: varchar(1028)

```

Comment:
is it safe to update existing Liquibase migration?

doesn't it throw exception in such case?